docs/Local-Development.md

```diff
@@ -0,0 +1,25 @@
+# Local Development of Inji Certify
+
+**Pre-requisites**: Java 21 installed, Postgres DB installed & configured
+
+1. Clone the repo, usually the active development happens on the `develop` branch but one can try out one of the `release-x.y.z` branches as well.
+2. Run the DB init scripts present in `db_scripts/mosip_certify` , running `deploy.sh deploy.properties` is a good way to init the DB.
+3. Decide on the issuance mode of Certify. Some plugins enable Certify to operate as a Proxy and others enable it to work as an Issuer, configure `mosip.certify.issuer` appropriately as `PluginIssuer` or `CertifyIssuer`.
+    * If you don't have another Issuance module such as Sunbird, MOSIP Stack, you may want to set it up with CertifyIssuer and configure `mosip.certify.issuer.vc-sign-algo`, `mosip.certify.issuer.pub.key`, `mosip.certify.issuer.uri` appropriately.
+4. Decide on the VCI plugin for use locally and configure it, while running locally from an IDE such as Eclipse or IntelliJ one needs to add configuration to the `application-local.properties` and add the VCI plugin dependency to the pom.xml of Certify Service.
+5. Get an eSignet 1.4.1 setup running configured with the appropriate Authenticator plugin implementation matching the VCI plugin.
```

Comment:
lets not call out the version specifically, then this need to kept updated.

Reply:
This may lead external users to fallback to the latest or older version of eSignet and spending time debugging eSignet. I know Certify is supposed to be independent of AuthZ provider, but can we add a compatibility matrix somewhere?